packages/serve/src/createConfig.ts

```diff
@@ -10,6 +10,15 @@ import { devServerOptionsType } from './types';
  */
 export default function createConfig(args): devServerOptionsType {
     const options = { ...args };
+    let isDevServer4 = false;
+    try {
+        // eslint-disable-next-line node/no-extraneous-require
+        const version = require('webpack-dev-server/package.json').version;
+
+        isDevServer4 = version.startsWith('4');
```

Comment:
Let's move out it from `try`/`catch`, because it is unnecessary have here

Reply:
done